e2e/js/src/js.test.ts

```diff
@@ -70,6 +83,13 @@ describe('js e2e', () => {
       `dist/libs/${lib}/src/lib/${lib}.d.ts`
     );
 
+    // Check that only react is added as a dep and no lockfile present
+    expect(readJson(`dist/libs/${lib}/package.json`).peerDependencies).toEqual({
```

Comment:
Sidenote: should this have a `package.json` if it's only buildable and not publishable?

Reply:
Yeah we generate the package.json file because the package can still be used outside of the workspace (like orgs copy tarballs).